gis4wrf/plugin/ui/widget_geo.py

```diff
@@ -59,11 +59,16 @@ def populate_tree(self) -> None:
             item.setText(1, description)
             item.setText(2, str(resolution))
             item.setFlags(item.flags() | Qt.ItemIsUserCheckable)
+            item.setToolTip(1, description)
+            if isinstance(resolution, str):
+                item.setToolTip(2, resolution)
+            else:
+                item.setToolTip(2, str(dd_to_dms(resolution)))
```

Comment:
That's only have the story. It should display the proper string as in the data tab.